room/templates/stafftime_static.html

```diff
@@ -65,7 +65,9 @@
     Description
   </td>
   <td>
-    {{ entity.description|escape|replace('\n', '<br>\n') }}
```

Comment:
We do have to worry about HTML escaping in these templates.  For example what if someone enters a description like "I am full of mischief <script> ... do something terrible .."

See this about the autoescaping in jinja2.   
 http://jinja.pocoo.org/docs/2.9/api/#autoescaping

I'm not sure if it's enabled.  It probably should be.